server/src/main/java/org/elasticsearch/cluster/routing/ShardRouting.java

```diff
@@ -855,4 +894,66 @@ public long getExpectedShardSize() {
     public RecoverySource recoverySource() {
         return recoverySource;
     }
+
+    public Role getRole() {
+        return role;
+    }
```

Comment:
Most of the accessors are named without `get` prefix in this class. Consider renaming to `role()`

Reply:
:+1: see [36e758f](https://github.com/elastic/elasticsearch/pull/92668/commits/36e758fcf96229d8a0996f5ed65c5eea2404add6)